docs/contributing/development.md

```diff
@@ -159,3 +159,29 @@ the latest changes from the master branch.
 ❯ git remote add upstream https://github.com/neutrinojs/neutrino.git
 ❯ git pull upstream master
 ```
+
+## Releasing a new version
+
+1. Decide whether the new version should be a major/minor/patch/prerelease version bump.
+2. From the root of the Neutrino repository, run: `git checkout -b version-bump master`
```

Comment:
Can we make this `git checkout -b version-bump upstream/master` to ensure we are using this from upstream or does that not matter? Also should we pair this with:

```sh
git fetch upstream --quiet && git checkout -b version-bump upstream/master
```

to ensure master is up to date?

Reply:
Good idea :-)